CONTRIBUTING.md

```diff
@@ -38,19 +38,27 @@ The preferred workflow for contributing to PyMC3 is to fork the [GitHub reposito
 3. Create a ``feature`` branch to hold your development changes:
 
    ```bash
-   $ git checkout -b my-feature
+   $ git switch -c my-feature
```

Comment:
git's new way of switching branches - `git checkout` still works, but this is meant to be more intuitive to beginners

Reply:
and this creates a new branch too?

Reply:
yup, `-c` for "create"